### DISCUSSION TITLE
[Feature] Mobile Turkish language support

### DISCUSSION THREAD

**Discussion by caglaryalcin** [immich-app/immich#13202] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Turkish language support is available in the Immich web interface. I request Turkish language support to be added on the mobile application. I would like to fork this and add it myself, but I could not find the appropriate file.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#12727
